### PR TITLE
WIP: Portals subdirectory

### PR DESCRIPTION
**Pull request by josenavas**
This branch holds the WIP to change the portals to work on subdirectories rather than in subdomain.

Remaining things to check:
 - [x] Uploader - when you upload files, it is trying to do it against the main site, rather than test
 - [x] Analysis - in the analysis pipeline, when moi creates the job group, something is acting up as it only creates the first job in the analysis (the creation one). All the links in moi look correct so I'm unsure where the source of the issue is in here.
 - [x] Once the study is created, the link that shows with "study BLA created" is linking to the main site rather than the portal.

### CODE REVIEW

File: qiita_core/configuration_manager.py [hunk 1]
```diff
@@ -162,6 +162,9 @@ def _get_main(self, config):
         self.max_upload_size = config.getint('main', 'MAX_UPLOAD_SIZE')
         self.require_approval = config.getboolean('main', 'REQUIRE_APPROVAL')
         self.portal = config.get('main', 'PORTAL')
```
Comment (squirrelo) [inline qiita_core/configuration_manager.py]: Both portal and portal_dir are missing from the docstring for the class.

File: qiita_core/support_files/config_test.cfg [hunk 2]
```diff
@@ -42,6 +42,9 @@ VALID_UPLOAD_EXTENSION = fastq,fastq.gz,txt,tsv,sff,fna,qual
 # Portal the site is working under
```
Comment (squirrelo) [inline qiita_core/support_files/config_test.cfg]: Since these have to do with portals, shouldn't these be under the portal section of the config file?

**Comment by squirrelo** [comment]
:+1:  for the config file updates

**Comment by antgonza** [comment]
:+1:

**Pull request merged**